### ISSUE TITLE
Bleeding Package Installation Error

### ISSUE THREAD

**Issue by rkm7232** [irontec/ivozprovider#1171]
### Description
Hi, I installed bleeding package on a debian machine to test some sort of features and evaluate the source code, When I login to web interface and going to 
Brand Configuration > Clients > [Any type of clients]
I get the following error:
**Module registration error.
Error: Undefined error.**

I know bleeding version is very unstable...
---

- IvozProvider version: 2.13~2.13.1~201910

- Related sections: 
    - [ ] SIP & Telephony
    - [ ] vPBX Functionalities
    - [ ] Billing & Invoicing
    - [x] Web Interface
    - [ ] API
    - [ ] Other

### Troubleshooting
Just install bleeding version from repo.

**Comment by gil-obradors** [comment]
Hi @rkm7232 ,
While waiting for colleagues with more knowledge than me...

Could you attach the log that is generated when the error appears. ( ```journalctl -f```  )

**Comment by rkm7232** [comment]
Thanks for your help. 

This is the exception:
Debian klear[7392]: Exception captured [500]: An exception occurred while executing 'SELECT c0_.type AS type_0, c0_.name AS name_1, c0_.domain_users AS domain_users_2, c0_.nif AS nif_3, c0_.distributeMethod AS distributeMethod_4, c0_.maxCalls AS maxCalls_5, c0_.maxDailyUsage AS maxDailyUsage_6, c0_.postalAddress AS postalAddress_7, c0_.postalCode AS postalCode_8, c0_.town AS town_9, c0_.province AS province_10, c0_.country AS country_11, c0_.ipFilter AS ipFilter_12, c0_.onDemandRecord AS onDemandRecord_13, c0_.allowRecordingRemoval AS allowRecordingRemoval_14, c0_.onDemandRecordCode AS onDemandRecordCode_15, c0_.externallyExtraOpts AS externallyExtraOpts_16, c0_.recordingsLimitMB AS recordingsLimitMB_17, c0_.recordingsLimitEmail AS recordingsLimitEmail_18, c0_.billingMethod AS billingMethod_19, c0_.balance AS balance_20, c0_.showInvoices AS showInvoices_21, c0_.id AS id_22, c0_.languageId AS languageId_23, c0_.mediaRelaySetsId AS mediaRelaySetsId_24, c0_.defaultTimezoneId AS defaultTimezoneId_25, c0_.brandId AS brandId_26, c0_.domainId AS domainId_27, c0_.applicationServerId AS applicationServerId_28, c0_.countryId AS countryId_29, c0_.currencyId AS currencyId_30, c0_.transformationRuleSetId AS transformationRuleSetId_31, c0_.outgoingDdiId AS outgoingDdiId_32, c0_.outgoingDdiRuleId AS outgoingDdiRuleId_33, c0_.vmNotificationTemplateId AS vmNotificationTemplateId_34, c0_.faxNotificationTemplateId AS faxNotificationTemplateId_35, c0_.invoiceNotificationTemplateId AS invoiceNotificationTemplateId_36, c0_.callCsvNotificationTemplateId AS callCsvNotificationTemplateId_37 FROM Companies c0_ WHERE c0_.type = ? AND c0_.brandId = ? ORDER BY c0_.name ASC LIMIT 25 OFFSET 0' with params ["vpbx", 1]:

                                    SQLSTATE[42S22]: Column not found: 1054 Unknown column 'c0_.maxDailyUsage' in 'field list' (0) # /opt/irontec/ivozprovider/library/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php (71)

maxdailyusage is a new feature and seems db column not created for it...

**Comment by Kaian** [comment]
Hi @rkm7232 

Looks like some migration have not been applied. ivozprovider-schema package [is in charge of applying new migrations](https://github.com/irontec/ivozprovider/blob/bleeding/debian/ivozprovider-schema.postinst#L56) to database

You could try running them manually:

```
/opt/irontec/ivozprovider/schema/bin/console doctrine:migrations:migrate -n
```

You can check the database status by running:
```
/opt/irontec/ivozprovider/schema/bin/console doctrine:migrations:status
```

Regards

**Comment by Kaian** [comment]
Migrations failing to apply may be related to bug solved by #1170

**Comment by rkm7232** [comment]
Yeah, some migrations failed to apply.

**Comment by Kaian** [comment]
Can you provide the APT installation log to determine the cause of failed migrations?

You can usually get it from /var/log/apt/term.log

**Comment by rkm7232** [comment]
Yeah, sure.

You can find it [https://pastebin.com/NPpkgH0B](here).

**Comment by Kaian** [comment]
Cool, thats totally the error that #1170 pretended to fix so this is a duplicate from #1141

Thanks a lot for reporting!!
